### PR TITLE
Update AMQ images to strict tags and remove AMQ Online 1.3.1 manifest

### PR DESCRIPTION
**Pull request by davidffrench**
# Description
https://issues.redhat.com/browse/INTLY-7181

- Updating AMQ images to strict tags, taken from https://brewweb.engineering.redhat.com/brew/builds?tagID=62779 and https://catalog.redhat.com/software/containers/search?q=amq&p=1&rows=30
- Removing 1.3.1 manifest

@mikenairn What would your preferred process be, the base branch for this PR is `amq-online-next-ga` as I thought the diff would be easier to read. I will then use `amq-online-next-ga` to merge into master and have full verification steps. Are you ok with this?

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

**Comment by mikenairn** [comment]
@davidffrench Yeah i think we will just treat these `*next` branches like master and PR changes into them and changes to master should always be applied from the delorean managed branch.

We will be creating some documentation around this next sprint.

The tests also won't currently run on these PR's.

File: manifests/integreatly-amq-online/amq-online-1.4.1/amq-online.1.4.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -630,65 +630,65 @@ spec:
...
+                  value: "registry.redhat.io/amq7/amq-online-1-console-server-rhel7:1.4-28"
                 - name: CONSOLE_PROXY_OPENSHIFT_IMAGE
                   value: "registry.redhat.io/openshift3/oauth-proxy:latest"
                 - name: CONSOLE_PROXY_KUBERNETES_IMAGE
                   value: "quay.io/pusher/oauth2_proxy:latest"
                 - name: "FS_GROUP_FALLBACK_MAP"
                   value: "{}"
                 - name: ENABLE_MONITORING
-                  value: "true"
+                  value: "false"
```
Comment (mikenairn) [inline manifests/integreatly-amq-online/amq-online-1.4.1/amq-online.1.4.1.clusterserviceversion.yaml:691]: Is this something we always need to do for amq-online?

**Comment by mikenairn** [comment]
re. Deleting the 1.3.1 CSV, does it matter that the 1.4.0 CSV is now referencing a CSV that no longer exists in this directory https://github.com/integr8ly/integreatly-operator/blob/amq-online-next-ga/manifests/integreatly-amq-online/amq-online-1.4.0/amq-online.1.4.0.clusterserviceversion.yaml#L230

File: manifests/integreatly-amq-online/amq-online-1.4.1/amq-online.1.4.1.clusterserviceversion.yaml (hunk 1 above)
Comment (davidffrench) [reply manifests/integreatly-amq-online/amq-online-1.4.1/amq-online.1.4.1.clusterserviceversion.yaml:691]: Good question @mikenairn , No, it is just until https://issues.redhat.com/browse/INTLY-5111 is done

**Comment by davidffrench** [comment]
> re. Deleting the 1.3.1 CSV, does it matter that the 1.4.0 CSV is now referencing a CSV that no longer exists in this directory https://github.com/integr8ly/integreatly-operator/blob/amq-online-next-ga/manifests/integreatly-amq-online/amq-online-1.4.0/amq-online.1.4.0.clusterserviceversion.yaml#L230

This is a good question, I am not sure what happens in this case. @philbrookes Do you know?

File: manifests/integreatly-amq-online/amq-online-1.4.1/amq-online.1.4.1.clusterserviceversion.yaml (hunk 1 above)
Comment (mikenairn) [reply manifests/integreatly-amq-online/amq-online-1.4.1/amq-online.1.4.1.clusterserviceversion.yaml:691]: OK, that's fine, i just want to make sure anything that can be automated is automated. Nothing to be done here if it's temporary.

**Comment by mikenairn** [comment]
/lgtm

@davidffrench I will leave it to you to approve if you think it's ready to merge. I'd imagine if the 1.3.1 thing is an issue the e2e tests should fail.

**Comment by davidffrench** [comment]
> /lgtm
> 
> @davidffrench I will leave it to you to approve if you think it's ready to merge. I'd imagine if the 1.3.1 thing is an issue the e2e tests should fail.

I sent a message on the `forum-operator-fw` slack channel to confirm. Here is the response. 
> the replaces field is used to build the registry's update graph. if a CSV specifies a replaces that doesn't exist the registry will fail to serve the entire operator because the package's update graph is invalid. so a subscription just won't be able to use it because it won't be visible to OLM at all

So I will update this PR to remove this. If we are looking for delorean to automate the removal of previous CSVs, it would need to modify any CSVs that have a replaces pointing to this as well.

**Comment by mikenairn** [comment]
> So I will update this PR to remove this. If we are looking for delorean to automate the removal of previous CSVs, it would need to modify any CSVs that have a replaces pointing to this as well.

@davidffrench Yeah i thought that might be the case. We will indeed be looking to automate this to remove the old CVSs once a new one comes in so will make sure this is always removed on the last one :+1: 

/lgtm

**Comment by davidffrench** [comment]
Thanks @mikenairn

**Pull request merged**